packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -688,6 +688,9 @@ const VideoPlayerBase = class extends React.Component {
 
 	componentDidMount () {
 		on('mousemove', this.activityDetected);
+		if (platform.touch) {
+			on('touchmove', this.activityDetected);
```

Comment:
I wonder if touching and releasing in the same spot over and over would be activity?  :D

Reply:
I believe the `RootContainer`'s `onClick` event handler will capture all tap events so it will be redundant to handle `touchstart`

Reply:
Yes, I realized that.  :P  It just seemed funny to me.